### PR TITLE
chore(content/update): Add Patch Level Tomcat 10 Support

### PR DESCRIPTION
**Pull request by psavidis**
Related-to: https://github.com/camunda/camunda-bpm-platform/issues/4391

### CODE REVIEW

**Review by danielkelemen — APPROVED** [review]

**Comment by psavidis** [comment]
> ### TODO
> **Action:** Finalise (update or keep as-is) the documentation for the `7.21` backport **When:** After the finalisation of **Patch Level Update** section of master **References:** [7.22 Pull Request](https://github.com/camunda/camunda-docs-manual/pull/1647#issuecomment-2200306986)

**Status:** **Done**
**References:** [Add Tomcat Manual Installation Section](https://github.com/camunda/camunda-docs-manual/pull/1660/commits/d018c0a30ea53003c622bc2d909b1403b3739b6c), [Add Patch Level Update Section](https://github.com/camunda/camunda-docs-manual/pull/1660/commits/812fa18f0ecdd505d0bbe351c2bfe13191583e88)

File: content/update/patch-level.md [hunk 1]
```diff
@@ -486,6 +486,21 @@ This will allow a better experience when looking for process definitions.
...
+This version supports all the necessary building-block modules for our users to use `camunda-bpm-platform` community and enterprise editions in conjunction with `Tomcat 10.1`.
+
+{{< note title="Heads-up!" class="warning" >}}
+`Tomcat 10` is compatible with the `jakarta` namespace. 
+If you wish to use it, the `jakarta` modules needs to be used (`camunda-webapp-tomcat-jakarta`, `camunda-engine-rest-jakarta`).
+
+The `javax` modules won't work with `Tomcat 10`.
+
+For detailed steps of manual installation, follow the [Tomcat Manual Installation Guide]({{< ref "/installation/full/tomcat/manual.md" >}})
+{{< /note >}}
```
Comment (tasso94) [inline content/update/patch-level.md:502]: ❌ Let's add the section from the 7.21 to 7.22 migration guide here as well:
* Migrate process applications
* Migrate Java webapp plugins

**Review by tasso94 — CHANGES_REQUESTED** [review]

File: content/update/patch-level.md (hunk 1 above)
Comment (psavidis) [reply content/update/patch-level.md:502]: Added both sections from the `7.21 to 7.22` migration guide to the Patch Level Section of the backport to make it more informative.

**Review by tasso94 — APPROVED** [review]

**Comment by psavidis** [comment]
### Update
Review completed, moving on with merging.

**Pull request merged**